code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -192,13 +195,13 @@
 	M.dizziness = max(0,M.dizziness-2)
 	M.drowsyness = max(0,M.drowsyness-1)
 	M.jitteriness = max(0,M.jitteriness-3)
-	M.AdjustSleeping(-1)
+	M.AdjustSleeping(-1, 0)
 	if(M.getToxLoss() && prob(20))
-		M.adjustToxLoss(-1)
+		M.adjustToxLoss(-1, 0)
 	if (M.bodytemperature < 310)  //310 is the normal bodytemp. 310.055
 		M.bodytemperature = min(310, M.bodytemperature + (20 * TEMPERATURE_DAMAGE_COEFFICIENT))
 	..()
-	return
+	. = 1
```

Comment:
The `.=1` can be under or above the `..()`, it doesn't change anything. the proc always requires an update so we need to always make it return 1.